core/arch/arm/plat-totalcompute/tc0_spmc_pm.c

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, Arm Limited
+ */
+
+#include <arm.h>
+#include <kernel/boot.h>
+#include <ffa.h>
+#include <sm/psci.h>
+
+void ffa_secondary_cpu_ep_register(vaddr_t secondary_ep)
+{
+	unsigned int ret = 0;
+
+	/* Invoke FFA_SECONDARY_EP_REGISTER_64 to the SPMC */
+	ret = thread_smc(FFA_SECONDARY_EP_REGISTER_64, secondary_ep, 0, 0);
+
+	if (ret != FFA_SUCCESS_32) {
```

Comment:
Perhaps we'd better check for both `FFA_SUCCESS_32` and `FFA_SUCCESS_64` since both a valid successful return values.

Reply:
Resolved.